packages/ChangesReporting/Output/ConsoleOutputFormatter.php

```diff
@@ -88,7 +89,7 @@ private function reportFileDiffs(array $fileDiffs): void
 
             $this->rectorOutputStyle->writeln($message);
             $this->rectorOutputStyle->newLine();
-            $this->rectorOutputStyle->writeln($fileDiff->getDiffConsoleFormatted());
```

Comment:
I think the difference between this PR and maste is, that in the PR we take `no-ansi` into account and use a different FileDiffer.

before this PR, we always printed the diffConsoleFormatted no matter `--no-ansi`.